### PR TITLE
OpenSnowMap

### PR DESCRIPTION
**Pull request by stell**
BaseLayer is restricted access but the pistes Overlay is great.

### CODE REVIEW

File: leaflet-providers.js [hunk 1]
```diff
@@ -1043,6 +1043,17 @@
 				detectRetina: true,
 				subdomains: '12'
 			}
+		},
+		OpenSnowMap: {
+			url: 'https://tiles.opensnowmap.org/{variant}/{z}/{x}/{y}.png',
+			options: {
+				minZoom: 9,
+				maxZoom: 18,
+				attribution: 'Map data: {attribution.OpenStreetMap} | contributors ODBL and www.opensnowmap.org CC-BY-SA'
```
Comment (brunob) [inline leaflet-providers.js:1052]: It would be nice to put a link on www.opensnowmap.org url ?

**Review by brunob — CHANGES_REQUESTED** [review]

**Comment by brunob** [comment]
@jieter any thoughts ?

**Comment by jieter** [comment]
Looks good, nice cleanup too!

**Comment by brunob** [comment]
thx @stell for the PR & @jieter for the feedback, i'll put this one in https://github.com/leaflet-extras/leaflet-providers/blob/master/preview/preview.js

**Pull request merged**